ovh/__init__.py,sha256=7js2ks483jKHVGfk7e1XDiGXauGy-53ixEkwFrFMp4g,1952
ovh/client.py,sha256=vUi3gwmSp5Ks8rIarv1Z7vvLzUIHtS9ZEhXDARVg8Ng,21281
ovh/config.py,sha256=9kgxEklERKZ_33zZYICPooofUXHZVMcWWngQO1yzsC0,4399
ovh/consumer_key.py,sha256=hoOCk3TuEMDXTNwjnAbPEyYT7S4SLzk27M87gQ-se4Q,4736
ovh/exceptions.py,sha256=iovxULysdcAI9mlyN-VwMur288A1I2qK8iirx05QeQ8,3561
ovh-1.1.0.dist-info/LICENSE,sha256=cxtKjkb1RJdlDtpp307Qb9YS7M9rfPDf6vbfWSKZ3bk,1468
ovh-1.1.0.dist-info/METADATA,sha256=SRnaPSav7CDzKGiiU1HIvbYBcN0zy9dlFlyq3Miv23c,21356
ovh-1.1.0.dist-info/WHEEL,sha256=a-zpFRIJzOq5QfuhBzbhiA1eHTzNCJn8OdRvhdNX0Rk,110
ovh-1.1.0.dist-info/top_level.txt,sha256=5g055HyiJp5qs7BUVRhkbWkZnzlPmWDZAwbbmaYSIwI,4
ovh-1.1.0.dist-info/RECORD,,
